source/_docs/team-management.md

```diff
@@ -1,34 +1,34 @@
 ---
-title: Team Management
+title: Site Team Management
 description: Working with the Pantheon Website Management Platform deployment tools in a team driven environment.
 categories: [managing]
 tags: [platform]
 keywords: team management, roles, permissions, team members, developer, privileges, change site owner, ownership, site owner, organization, supporting organization, agency, supporting agency
 ---
 Pantheon has powerful workflow tools that are packed with real-time features that are great for people working in teams, and getting started is easy.
 
-## Team Management
-You can add and delete users in the Team modal by clicking **Team** in the upper-right of your Dashboard.
+## Site Team Management
+You can add and delete users in the Team modal by clicking **Team** in the upper-right of your Site Dashboard.
 
 ![Team Management Window](/source/docs/assets/images/team-modal.png)
-## Add a Team Member
+## Add a Site Team Member
```

Comment:
The title of the doc makes it clear this is managing site team members, so let's keep the subheaders as they are.